content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,571 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Kubernetes
 
-This pack defines the properties that will be used by Spectro Cloud to bring up the Kubernetes cluster. Most of the Kubernetes hardening standards recommended by the Center for Internet Security is enabled by default.
+The Kubernetes pack defines the properties that Spectro Cloud uses to deploy Kubernetes clusters. Most of the Kubernetes hardening standards that the Center for Internet Security (CIS) recommends are enabled by default. The Kubernetes pack supports most [cloud and data center infrastructure providers](/clusters).
 
-<WarningBox>
-Info: Only Kubernetes versions 1.19 and above is supported
-</WarningBox>
+Spectro Cloud offers Cloud Native Computing Foundation (CNCF) Kubernetes as a core infrastructure pack in Palette. We release Kubernetes updates as follows:
+
+<br />
+
+- Major versions are assessed based on the extent of changes.
+- Minor versions are provided within eight weeks of a new Kubernetes release.
+- Patch versions are provided within four weeks of a new Kubernetes release.
 
-## Supported Kubernetes versions
+In the period of time following a Kubernetes release, the Kubernetes pack undergoes rigorous vulnerability scans and penetration testing. 
 
-<InfoBox>
+A minor Kubernetes version will be deprecated in Palette when the Kubernetes community announces the version is entering End of Life (EOL). The deprecated version will remain in that state for three months and will be disabled after a quarter and deleted from Palette in the following quarter.
 
-Versions supported in the latest [release](/release-notes/) are highlighted.
+<br />
+
+## Versions Supported
 
-</InfoBox>
 
 <Tabs>
 
-<Tabs.TabPane tab="1.19.x" key="k8s_v1.19">
-
-- **1.19.16**
-- **1.19.15**
-- **1.19.14**
-- **1.19.13**
-- **1.19.12**
-- **1.19.11**
-- **1.19.10**
-- **1.19.9**
-- **1.19.8**
-- **1.19.7**
-- **1.19.6**
-- **1.19.5**
-- **1.19.4**
-- **1.19.3**
+<Tabs.TabPane tab="1.26.x" key="k8s_v1.26">
 
-</Tabs.TabPane>
+## Prerequisites
 
-<Tabs.TabPane tab="1.20.x" key="k8s_v1.20">
-
-- **1.20.12**
-- **1.20.11**
-- **1.20.10**
-- **1.20.9**
-- **1.20.8**
-- **1.20.7**
-- **1.20.6**
-- **1.20.5**
-- **1.20.4**
-- **1.20.2**
-- **1.20.1**
-- **1.20.0**
+- A minimum of 4 CPU and 4GB Memory.
 
-</Tabs.TabPane>
+- Operating System (OS) dependencies as listed in the table.
 
-<Tabs.TabPane tab="1.21.x" key="k8s_v1.21">
 
-- **1.21.10**
-- **1.21.6**
-- **1.21.5**
-- **1.21.4**
-- **1.21.3**
-- **1.21.2**
-- **1.21.1**
-- **1.21.0**
+| Operating System | Kubernetes 1.26.x |
```

Comment:
This could be me, but it was not obvious what the X represented. I'm guessing it means "not available"? It will help if we explain the table a bit.  In the second column of the table, I wonder if we state "Support Enabled" could clarify the confusion. 

We could also use these icons ✅ and ❌ to improve readability.  If you want to use those icons, you literally can copy and paste them to the markdown 😄

Reply:
Discussed with Karl and implemented his proposed table.